# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1217117335 -7200
#      Sun Jul 27 02:08:55 2008 +0200
# Node ID b224a9a9ccf1bc2c2c4f4fd44cd1be91a3cddcbc
# Parent  ff6c1534b522b7199b326bb1183b8fe6934f6e00
Calrify OuterPosition property in manual

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2008-07-28  David Bateman  <dbateman@free.fr>
+
+	* interpreter/plot.txi: Clarify the description of OuterPosition
+	axes property.
+
 2008-05-20  David Bateman  <dbateman@free.fr>
 
 	* interpreter/debug.txi: Uodate documentation for multiline
 	input. Add documentation for dbcont, dbquit, dbstep, dbstqck, dbup
 	and dbdown functions.
 	* interpreter/octave.texi: Upodate index for debugging functions.
 
 2008-05-03  Rafael Laboissiere <rafael@debian.org>
diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -745,20 +745,20 @@ May be one of
 @item "new"
 @item "add"
 @item "replace"
 @item "replacechildren"
 @end table
 
 @item outerposition
 A four-element vector specifying the coordinates of the lower left
-corner and width and height of the plot, in normalized units.  For
-example, @code{[0.2, 0.3, 0.4, 0.5]} sets the lower left corner of the
-axes at @math{(0.2, 0.3)} and the width and height to be 0.4 and 0.5
-respectively.
+corner and width and height of the plot, in normalized units including
+the tics, labels etc.  For example, @code{[0.2, 0.3, 0.4, 0.5]} sets the
+lower left corner of the axes at @math{(0.2, 0.3)} and the width and
+height to be 0.4 and 0.5 respectively.
 @end table
 
 @node Line Properties
 @subsubsection Line Properties
 
 @table @code
 @itemx xdata
 @itemx ydata
